client/gutenberg-editor/index.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { siteSelection, sites } from 'my-sites/controller';
+import { post } from './controller';
+import config from 'config';
+import { makeLayout, render as clientRender } from 'controller';
+
+export default function() {
+	if ( config.isEnabled( 'gutenberg' ) ) {
+		page( '/gutenberg', () => page.redirect( '/gutenberg/post' ) );
+		page( '/gutenberg/post', siteSelection, sites, makeLayout, clientRender );
+		page( '/gutenberg/post/:site?/:post?', siteSelection, post, makeLayout, clientRender );
+		page( '/gutenberg/page', siteSelection, sites, makeLayout, clientRender );
+		page( '/gutenberg/page/:site?/:post?', siteSelection, post, makeLayout, clientRender );
+	} else {
+		page( '/gutenberg', () => page.redirect( '/post' ) );
+		page( '/gutenberg/*', () => page.redirect( '/post' ) );
```

Comment:
Nit: redirect can be written as `page( '/gutenberg', '/post' )`.

Reply:
Good catch, that was a copy pasta on my part. Updated in 76d3e27